nucleus/validate/client.py

```diff
@@ -78,6 +83,30 @@ def create_scenario_test(
                 "Must pass an evaluation_criteria to the scenario test! I.e. "
                 "evaluation_criteria = [client.validate.eval_functions.bbox_iou() > 0.5]"
             )
+        incorrect_type = [
```

Comment:
More concrete error message for (the probably common case) of forgetting to add a comparison operator to the `evaluation_criteria`